docs/index.rst

```diff
@@ -104,6 +104,22 @@ A Glance:
     u'The study shows that 93% of people ...'
 
 
+Newspaper can directly parse a string:
+
+.. code-block:: pycon
+
+    >>> from newspaper import Article
+
+    >>> a = Article('http://example.com/fake')  # Pass any url
```

Comment:
I'm not sure I would be able to understand this without having context. Can you expand upon it just a little? 
> Pass any url, valid or not. This will be overridden in the next steps.